docs/userGuide/siteConfiguration.md

```diff
@@ -29,7 +29,10 @@ Here is a typical `site.json` file:
       "searchable": "no",
       "externalScripts": [
         "https://cdn.plot.ly/plotly-latest.min.js"
-      ]
+      ],
+      "frontmatter": {
+        "header": "header.md"
+      }
```

Comment:
This part is in the wrong place? Should be inside the specified page.

Reply:
Sorry, I'm not sure I get the issue... It seems correct to me

You mean inside the file? We support specifying them in `pages` as well

Reply:
Sorry, my bad, read this part too quickly and got confused, thought it was specified outside of `pages`...